console/console-init/ui/src/graphql-module/queries/project.ts

```diff
@@ -91,4 +137,42 @@ const RETURN_ALL_PROJECTS = (
   return ALL_PROJECTS;
 };
 
-export { RETURN_ALL_PROJECTS };
+const RETURN_ALL_PROJECTS_FOR_NAME_OR_NAMESPACE = (
+  propertyName: string,
+  value: string
+) => {
+  let filter = "";
+  value = removeForbiddenChars(value);
+  if (value && propertyName) {
+    filter += "`$.metadata." + [propertyName] + "` LIKE '" + value + "%'";
+  }
+
+  const all_proejcts = gql(`
+    query all_projects {
+      allProjects(filter: "${filter}" first:100 offset:0) {
```

Comment:
why do you have first and offset values hardcode ?

Reply:
Because for typeahead we fetch top 100 records.